committeeoversightapp/templates/committeeoversightapp/landing_page.html

```diff
@@ -1,10 +1,82 @@
 {% extends "base.html" %}
+{% load wagtailcore_tags %}
+{% load committeeoversight_tags %}
 
 {% block content %}
-  <div class="col-12 col-lg-9 mx-auto">
-    <h1 class="mb-4">{{page.title}}</h1>
+
+  <div class="row">
+    <div class="col-12 col-lg-4 col-xl-4 landing-description">
+        <h1 class="mb-4 ml-3 d-minus-rating">{{page.title}}</h1>
+        <div class="m-3">{{page.body|richtext}}</div>
+        <label class="m-3">
+          <h4 class="mb-3">Search by committee:</h4>
+          <select id="committee-select" class="form-control select2-single">
+            {% for committee in committees %}
+              <option value="{{committee.extras|lookup:"cleaned_id"}}">{{committee.name}}</option>
```

Comment:
i believe you can get at dictionary keys in a template using `.` notation, e.g., `dict.key`.

Reply:
This thread said no but I now realize it's 7 years old! https://stackoverflow.com/questions/8000022/django-template-how-to-look-up-a-dictionary-value-with-a-variable

I'll wait and see how https://github.com/datamade/committee-oversight/pull/82#discussion_r326280287 shakes out before making any changes here.